cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/UpdateController.java

```diff
@@ -62,17 +65,41 @@ public Integer call() throws Exception {
         }
     }
 
-    @Command(name = "remove-service-instance", description = "Removes a service instance.")
+    @Command(name = "remove-service", description = "Removes a service instance.")
     static class RemoveServiceInstanceCommand implements Callable<Integer> {
 
         @Mixin
         LoginCommandOptions loginOptions;
 
         @Mixin
         YamlCommandOptions yamlCommandOptions;
+      
+        @Option(names = { "-f", "--force" }, required = false, description = "Force deletion without confirmation.")
+        Boolean force;
 
         @Override
         public Integer call() throws Exception {
+
+            if (force != null) {
+                doRemoveServiceInstance(yamlCommandOptions);
+
+            } else {
+                
+                System.out.println("Really delete the service y/n?");
+                String input = new Scanner(System.in).nextLine();
+                
+                if (input.equals("y") || input.equals("yes")) {
+                    doRemoveServiceInstance(yamlCommandOptions);
+                } else {
+                    System.out.println("Delete cancelled");
+                    return 0;
```

Comment:
This should be considered an error condition, please return a value `>0` (e.g., `1`).